doc/whats-new.rst

```diff
@@ -23,6 +23,10 @@ v0.16.2 (unreleased)
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
+- :py:attr:`~core.accessor_dt.DatetimeAccessor.weekofyear` and :py:attr:`~core.accessor_dt.DatetimeAccessor.week`
+  have been deprecated. Use ``DataArray.dt.isocalendar().week``
```

Comment:
This should now be verbatim, right? Anyone an idea why only `week` is displayed on RTD?

https://xray--4534.org.readthedocs.build/en/4534/whats-new.html

Reply:
It should, but I don't know why it doesn't. Maybe try rerunning the CI by closing and reopening?